onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -286,9 +299,17 @@ def run_tensorflow(use_gpu, model_names, model_class, precision, batch_sizes, se
                 input_ids = tf.constant(values, shape=(batch_size, sequence_length), dtype=tf.int32)
 
                 try:
-                    model(input_ids, training=False)
+                    def encoder_forward():
+                        return model(input_ids, training=False)
+
+                    def encoder_decoder_forward():
+                        return model(input_ids, decoder_input_ids=input_ids, training=False)
```

Comment:
I think transformers has changed to make decoder_input_ids optional:
https://github.com/huggingface/transformers/commit/6912265711fdf29e537e35194552bfa2c31b72bc
Please install transformers from source and try it.

Reply:
it looks like current master of transformers can work without current change for torchscript benchmark

Reply:
I suggest we wait for next release of transformers package. They usually have short release interval (2~4 weeks).

Reply:
ok